### ISSUE TITLE
What is the main class right now?

### ISSUE THREAD

**Issue by blublinsky** [jvm-operators/abstract-operator#56]
In the latest version, class SDKEntrypoint replacing EntryPoint does not have a main method now.
Do I need to implement my own main class, that invokes SDKEntrypoint?
Is there example of such main somewhere?

**Comment by jkremser** [comment]
Yeah, I need to update the docs. There is basically no `public static void main`. The app's entrypoint is defined using the CDI events.
example: 
https://github.com/jvm-operators/abstract-operator/blob/master/src/main/java/io/radanalytics/operator/SDKEntrypoint.java#L83

`onStart(@Observes StartupEvent event)  ~ main`

check also: https://jkremser.github.io/post/spark-operator-quarkus/#static-initializers
